docs/api/input.md

```diff
@@ -66,16 +68,16 @@ import Fill from '@site/static/usage/v7/input/fill/index.md';
 
 ## Helper & Error Text
 
-Helper & error text can be used inside of an item with an input by slotting a note in the `"helper"` and `"error"` slots. The error slot will not be displayed unless the `ion-invalid` class is added to the `ion-item`. In Angular, this is done automatically through form validation. In JavaScript, React and Vue, the class needs to be manually added based on your own validation.
+Helper and error text can be used inside of an input with the `helperText` and `errorText` property. The error text will not be displayed unless the `ion-invalid` class is added to the `ion-input`. In Angular, this is done automatically through form validation. In JavaScript, React and Vue, the class needs to be manually added based on your own validation.
```

Comment:
This playground is reused from `ion-item`, so we'll need to create a new one to show the new syntax here.

Reply:
Good catch. I moved the playground from ion-item to ion-input and updated the syntax in https://github.com/ionic-team/ionic-docs/pull/2682/commits/65e5cdfbf19082d25875c0c163e25778bc703e8f and https://github.com/ionic-team/ionic-docs/pull/2682/commits/f09fbd800e939fb0d4eca29d4ac545bd88b8769f